src/__tests__/networks.integration.test.ts

```diff
@@ -216,8 +216,8 @@ describe('ethrResolver (alt-chains)', () => {
       const doc = await resolver.resolve(did)
       return expect(doc).toEqual({
         didDocumentMetadata: {
-          updated: '2022-01-19T12:19:59Z',
-          versionId: '57702193',
+          updated: '2022-01-19T12:20:00Z',
```

Comment:
The recent upgrade to the aurora network caused their "blocks" to be reorganized.
They claim to be EVM compatible now so it makes sense to use the new block order going forward.

This fixes the recent build failures